Src/EB/AMReX_EB_utils.cpp

```diff
@@ -406,6 +420,7 @@ void FillSignedDistance (MultiFab& mf, EB2::Level const& ls_lev,
     }
 
     mf.FillBoundary(0,1,ls_lev.Geom().periodicity());
+    mf_in.ParallelCopy(mf);
```

Comment:
mf_in may have ghost cells, but this version of  ParallelCopy does not involve ghost cells.